Resources/translations/SonataAdminBundle.pl.xliff

```diff
@@ -88,7 +88,7 @@
             </trans-unit>
             <trans-unit id="title_show">
                 <source>title_show</source>
-                <target>title_show</target>
+                <target>Szczegóły</target>
```

Comment:
I believe that this translation should contain a parameter , did you check that on the english version?